db/schema.rb

```diff
@@ -65,6 +69,7 @@
     t.string   "current_sign_in_ip",     limit: 255
     t.string   "last_sign_in_ip",        limit: 255
     t.boolean  "admin",                              default: false, null: false
+    t.datetime "created_at"
```

Comment:
Oh weird, I didn't notice that. I was doing some migrating and schema loading to resolve some issues with my development database, so that was probably the cause.